### PR TITLE
docs: Add Apache APISIX 3.2 release blog

### PR DESCRIPTION
**Pull request by KID-G**
Changes:

Add Apache APISIX 3.2 release blog zh

### CODE REVIEW

File: blog/zh/blog/2023/03/09/release-apache-apisix-3.1.0.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+            "variables": [
+                "name",
+                "githubAccount"
+            ]
+        }
+    }
+}'
+```
+
+这里 query 是我们要用到的查询语句，variables 是事先声明的变量列表。
```
Comment (SkyeYoung) [inline blog/zh/blog/2023/03/09/release-apache-apisix-3.1.0.md]:
```suggestion
这里 `query` 是我们要用到的查询语句，`variables` 是事先声明的变量列表。
```

**Review by SkyeYoung — CHANGES_REQUESTED** [review]

File: blog/zh/blog/2023/03/09/release-apache-apisix-3.1.0.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+
+```
+
+## 新插件：请求体/响应体转换
+
+你是否在为如何把返回 XML 的古早上游服务介绍给只接受 JSON 的现代客户端而苦恼？3.2.0 版本新增的 `body-transformer` 插件开源解决这个问题。
+
+`body-transformer` 插件支持 JSON 和 XML 之间的互相转换。不过这并非是它唯一能做的事。它还支持通过模板配置输入输出的内容的具体格式。举个例子，
+
+假设你有下面的 JSON 模板：`{"foo":"{{name .. " world"}}","bar":{{age+10}}}`，并把它配置到 body-transformer 插件的 `request.template` 字段中：
```
Comment (guoqqqi) [inline blog/zh/blog/2023/03/09/release-apache-apisix-3.1.0.md]:
```suggestion
假设你有下面的 JSON 模板：`{"foo":"{{name .. " world"}}","bar":{{age+10}}}`，并把它配置到 `body-transformer` 插件的 `request.template` 字段中：
```

File: blog/zh/blog/2023/03/09/release-apache-apisix-3.1.0.md [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+      ...
+      "body-transformer": {
+          "request": {
+              "template": "..."
+          }
+      }
+      ...
+```
+
+那么当请求内容为 `{"name":"hello","age":20}`，发送给上游的是改写之后的 `{"foo":"hello world","bar":30}`。我们采用了 lua-resty-template 来渲染模板，所以你可以在模板中嵌入 Lua 表达式来实现改写逻辑。
```
Comment (guoqqqi) [inline blog/zh/blog/2023/03/09/release-apache-apisix-3.1.0.md]:
```suggestion
那么当请求内容为 `{"name":"hello","age":20}`，发送给上游的是改写之后的 `{"foo":"hello world","bar":30}`。我们采用了 `lua-resty-template` 来渲染模板，所以你可以在模板中嵌入 Lua 表达式来实现改写逻辑。
```

File: blog/zh/blog/2023/03/09/release-apache-apisix-3.1.0.md [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+假设你有下面的 JSON 模板：`{"foo":"{{name .. " world"}}","bar":{{age+10}}}`，并把它配置到 body-transformer 插件的 `request.template` 字段中：
+
+```
+      ...
+      "body-transformer": {
+          "request": {
+              "template": "..."
+          }
+      }
+      ...
```
Comment (guoqqqi) [inline blog/zh/blog/2023/03/09/release-apache-apisix-3.1.0.md:211]: format

Comment (KID-G) [reply blog/zh/blog/2023/03/09/release-apache-apisix-3.1.0.md:211]: "body-transformer": {
    "request": {
        "template": "..."
    }
}

like this?

**Review by guoqqqi — DISMISSED** [review]

**Review by guoqqqi — APPROVED** [review]

**Review by Baoyuantop — APPROVED** [review]

**Review by SkyeYoung — APPROVED** [review]

**Pull request merged**